tempesta_fw/client.c

```diff
@@ -46,8 +46,6 @@ tfw_destroy_client(struct sock *s)
 	conn->hndl = NULL;
 
 	kmem_cache_free(cli_cache, cli);
-
-	conn->sk_destruct(s);
```

Comment:
I don't see any comment in the commit as to why this is incorrect and need to be removed.